src/screens/SignInV2/index.js

```diff
@@ -84,10 +90,16 @@ const SignIn = () => {
           removeLocalStorege('userId');
           navigation.dispatch(StackActions.replace('ChooseUsername'));
           setUserId(response?.data?.humanIdData?.appUserId);
+          AnalyticsEventTracking.eventTrack(
+            BetterSocialEventTracking.HUMAN_ID_SUCCESS_NEW_REGISTRATION
+          );
         }
       } catch (e) {
         SimpleToast.show(e?.message, SimpleToast.SHORT);
         crashlytics().recordError(new Error(e?.message));
+        AnalyticsEventTracking.eventTrack(BetterSocialEventTracking.HUMAN_ID_FAILED_VERIFICATION, {
+          error: e
+        });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a potential issue with the error handling here. The `e` object is being passed directly to the `eventTrack` function, which might cause issues if it contains non-serializable properties. It would be better to pass only the necessary information, such as the error message.
```diff
-          AnalyticsEventTracking.eventTrack(BetterSocialEventTracking.HUMAN_ID_FAILED_VERIFICATION, {
-            error: e
-          });
+          AnalyticsEventTracking.eventTrack(BetterSocialEventTracking.HUMAN_ID_FAILED_VERIFICATION, {
+            error: e.message
+          });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->